demos/storybook/stories/user-menu/with-custom-menu-header.stories.ts

```diff
@@ -0,0 +1,70 @@
+const Trex = require('../../assets/trex.png');
+const eatonLogo = require('../../assets/EatonLogo.svg');
+
+export const withCustomMenuHeader = (): any => ({
```

Comment:
The react SB calls this "with custom menu" and uses other components for the list items (instead of info-list-item)

Reply:
Good catch, I've reverted the story name and created the menu items using mat-list-items instead of info-list-items to showcase the difference.